broker/test/test_broker/jobs.BackupReaperJobs.spec.js

```diff
@@ -194,7 +194,8 @@ describe('Jobs', function () {
       // Mocks done
       getScheduleStub = sinon.stub(ScheduleManager, 'getSchedule').callsFake(getJob);
       return BackupReaperJob.run(job, () => {
-        mocks.verify();
+        let ignoreNock = ["POST https://myopenstackcloud.com:5000/v3/auth/tokens"];
```

Comment:
As the order of execution of UTs are changed, the token is not expired yet and so another authentication(login) is not required